### PR TITLE
Rewrite introduction for Sawtooth documentation

### PR DESCRIPTION
**Pull request by chenette**
Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: docs/source/introduction.rst [hunk 1]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
-    :doc:`PoET 1.0 Specification <../architecture/poet>`.
+Try Hyperledger Sawtooth
+------------------------
 
+The Sawtooth documentation explains how to set up a local validator for
+demonstrating Sawtooth functionality and for testing an application. Once
+running, you will be able to submit new transactions and fetch the resulting
+state and block data from the blockchain using HTTP and the Sawtooth REST API.
+These methods detailed apply to the included example transaction families, as
+well as to any transaction families you might write yourself.
```
Comment (aludvik) [inline docs/source/introduction.rst]: Typo `s/These/The/`

File: docs/source/introduction.rst [hunk 2]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
-============
+Develop a Custom Application
+----------------------------
 
-One repository contains all of the code needed:
+In Sawtooth, the data model and transaction language are implemented in a
+transaction family. Transaction families codify business rules used to modify
+state, while client programs typically submit transactions and view state. You
+can build custom transaction families that reflect the unique requirements of
+your ledgers, using the provide several core transaction families as models.
```
Comment (aludvik) [inline docs/source/introduction.rst]: typo `s/provide/provided/`, also remove 'several'

File: docs/source/introduction.rst [hunk 3]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    following items:
 
     * The implementation of the validator process which runs on each node
     * SDKs for writing transaction processing or validation logic in a variety
       of languages
     * Dockerfiles to support development or launching a network of validators
     * Source files for this documentation
 
+  * `Seth <https://github.com/hyperledger/sawtooth-seth>`_:
+    Deploy Ethereum Virtual Machine (EVM) smart to Sawtooth
```
Comment (aludvik) [inline docs/source/introduction.rst]: typo `s/smart/smart contracts/`

File: docs/source/introduction.rst (hunk 1 above)
Comment (chenette) [reply docs/source/introduction.rst]: Fixed with rewrite ("These methods apply...").

File: docs/source/introduction.rst (hunk 2 above)
Comment (chenette) [reply docs/source/introduction.rst]: Fixed.

**Comment by chenette** [comment]
Adding Shawn's comments from our meeting today.

File: docs/source/introduction.rst [hunk 4]
```diff
@@ -2,43 +2,162 @@
...
+elects a leader through some form of lottery, and variants of the traditional
+`Byzantine Fault Tolerance (BFT)
+<https://en.wikipedia.org/wiki/Byzantine_fault_tolerance>`_
+algorithms, which use multiple rounds of explicit votes to achieve consensus.
+
+Sawtooth abstracts the core concepts of consensus and isolates consensus from
+transaction semantics. The interface supports plugging in various consensus
+implementations. More importantly, Sawtooth allows different types of
+consensus on the same blockchain. The application can choose at compile time
+or change on a running blockchain with a transaction.
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change to:
   "The consensus is selected during the initial network setup and can be changed on a running blockchain with a transaction."

File: docs/source/introduction.rst [hunk 5]
```diff
@@ -2,43 +2,162 @@
...
+Sawtooth currently supports these consensus implementations:
+
+    * Proof of Elapsed Time (PoET), a Nakamoto-style consensus algorithm that is
+      designed to be a production-grade protocol capable of supporting large
+      network populations. PoET relies on secure instruction execution to
+      achieve the scaling benefits of a Nakamoto-style consensus algorithm
+      without the power consumption drawbacks of the Proof of Work algorithm.
+
+    * PoET simulator, which provides PoET-style
+      consensus on any type of hardware.
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Add:

     "including a virtualized cloud environment."

File: docs/source/introduction.rst [hunk 6]
```diff
@@ -2,43 +2,162 @@
...
+      developers and test networks that require only crash fault tolerance.
+
+Sample transaction families
+---------------------------
 
-Transaction Families
-====================
 In Sawtooth, the data model and transaction language are implemented
 in a *transaction family*. While we expect users to build custom transaction
 families that reflect the unique requirements of their ledgers, we provide
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Delete Validator Registry from the list below. It's really PoET Validator Registry, and is very specific to PoET.

File: docs/source/introduction.rst [hunk 7]
```diff
@@ -2,43 +2,162 @@
...
+Sawtooth provides smart contract abstraction that allows application
+developers to write contract logic in a language of their choice.
+
+An application can be a native business logic or a smart contract virtual
+machine. In fact, both types of applications can co-exist on the same
+blockchain. Sawtooth allows these design decisions to be made in the
+transaction-processing layer, which allows multiple types of applications to
+exist in the same instance of the blockchain network.
+
+Each application defines the custom transaction processors for the unique
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change to:

     "Each application defines custom transaction processors for its unique requirements."

File: docs/source/introduction.rst [hunk 8]
```diff
@@ -2,43 +2,162 @@
...
+requirements of their ledgers. Sawtooth provides several example transaction
+families to serve as models for low-level functions, such as registering
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change "registering ledger services" to:

     "maintaining chain-wide settings"

File: docs/source/introduction.rst [hunk 9]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+
+  * XO: Demonstrates how to construct basic transactions by playing
+    `Tic-tac-toe <https://en.wikipedia.org/wiki/Tic-tac-toe>`_. The
+    XO transaction family includes create and take transactions, with an XO
+    command-line interface (CLI) that allows two participants to play the game.
+    For more information, see
+    :doc:`/app_developers_guide/intro_xo_transaction_family`.
+
+  * Sawtooth Supply Chain:
+    Distributed application (DApp) to help you trace the provenance and other
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Don't mention DApps in this description.  Start with "Helps you trace..."

File: docs/source/introduction.rst [hunk 10]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    contextual information of any asset. Supply Chain provides a complete
+    example application with a transaction processor, custom REST API, and web
+    app. This DApp demonstrates a secure and decentralized solution for
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change to: "This application also demonstrates"

File: docs/source/introduction.rst [hunk 11]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    in-browser transaction signing. It also illustrates how to synchronize the
```
Comment (chenette) [inline docs/source/introduction.rst]: Drop the second "also".

File: docs/source/introduction.rst [hunk 12]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    blockchain state to a local database for complex queries. For more
+    information, see `<https://github.com/hyperledger/sawtooth-supply-chain>`_.
+
+  * Sawtooth Marketplace:
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Ask the Marketplace developers for a better description.

File: docs/source/introduction.rst [hunk 13]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    Blockchain application built on Hyperledger Sawtooth, allowing users to
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Drop first phrase; start with "Allows users to ..."

File: docs/source/introduction.rst [hunk 14]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    exchange quantities of customized "Assets" with other users on the
+    blockchain. This repo contains a number of components that, together with a
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change "repo" to "application".

File: docs/source/introduction.rst [hunk 15]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    Sawtooth validator, will run a Sawtooth blockchain and provide a simple
+    RESTful API to interact with it.  For more information, see
+    `<https://github.com/hyperledger/sawtooth-marketplace>`_.
+
+  * Sawtooth Private UTXO:
+    Experimental distributed application built on Hyperledger Sawtooth. This
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Drop first sentence.  Start with "Demonstrates how assets can be ..."

File: docs/source/introduction.rst [hunk 16]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
+    example demonstrates how assets can be created and traded on the Sawtooth
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Drop "on the Sawtooth distributed ledger".

File: docs/source/introduction.rst [hunk 17]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
-    supporting large network populations. Sawtooth includes an implementation
-    which simulates the secure instructions. This should make it easier for
-    the community to work with the software but also forgoes Byzantine fault
-    tolerance.  For more information, see
-    :doc:`PoET 1.0 Specification <../architecture/poet>`.
+Try Hyperledger Sawtooth
+------------------------
 
+The Sawtooth documentation explains how to set up a local validator for
+demonstrating Sawtooth functionality and for testing an application. Once
```
Comment (chenette) [inline docs/source/introduction.rst]: Drop the second "for".

File: docs/source/introduction.rst [hunk 18]
```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
...
-Repositories
-============
+Develop a Custom Application
+----------------------------
 
-One repository contains all of the code needed:
+In Sawtooth, the data model and transaction language are implemented in a
+transaction family. Transaction families codify business rules used to modify
+state, while client programs typically submit transactions and view state. You
+can build custom transaction families that reflect the unique requirements of
```
Comment (chenette) [inline docs/source/introduction.rst]: [ SHAWN ] Change to "custom transaction families that reflect your unique requirements."

File: docs/source/introduction.rst (hunk 6 above)
Comment (chenette) [reply docs/source/introduction.rst]: Done

**Pull request merged**